pkgs/data/fonts/corefonts/default.nix

```
@@ -30,46 +30,48 @@ stdenv.mkDerivation {
     inherit sha256;
   }) fonts;
 
-  nativeBuildInputs = [ cabextract ];
+  nativeBuildInputs = [ cabextract rename ];
 
   buildCommand = ''
     for i in $exes; do
       cabextract --lowercase $i
     done
     cabextract --lowercase viewer1.cab
+    # Avoid issues with case-insensitive file systems
+    rename 's/\.ttf$/.tt/' *.ttf
```

```suggestion
    rename 's/\.ttf$/.ttf.temp/' *.ttf
```
We should probably preserve the original extensions

---

Many thanks for the feedback! Have adopted your suggestion except have used `.tmp` instead of `.temp`.